packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -123,7 +123,7 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
           calloutProps={{ ...calloutProps }}
           styles={hostStyles}
         >
-          {strings.sendingAriaLabel && <LiveMessage message={strings.sendingAriaLabel} aria-live="polite" />}
+          {strings.sendingAriaLabel && <LiveMessage message={strings.sendingAriaLabel} ariaLive="polite" />}
```

Comment:
Small thing but did we do any announcer testing here? we have had bugs around this component in the past not announcing states

Reply:
Just the testing done here: https://github.com/Azure/communication-ui-library/pull/3005. Confirmed the DOM shows the same behavior as before